# LAPACK tests are failing on osx-arm64 due to some issues with gfortran 11.2
# when set to armv8.3-a. This should be fixed when gfortran gets updated to 11.3.
# In the meantime we can use this workaround:
# See https://github.com/xianyi/OpenBLAS/issues/3735
===================================================================
diff --git a/Makefile.arm64 b/Makefile.arm64
index 4efa5528..dff5c829 100644
--- a/Makefile.arm64
+++ b/Makefile.arm64
@@ -208,7 +208,7 @@ endif
 ifeq ($(CORE), VORTEX)
 CCOMMON_OPT += -march=armv8.3-a
 ifneq ($(F_COMPILER), NAG)
-FCOMMON_OPT += -march=armv8.3-a
+FCOMMON_OPT += -march=armv8-a
 endif
 endif
 
